docs/voldoen-aan-wetten-en-regels/hulpmiddelen/standaarden.md

```diff
@@ -0,0 +1,127 @@
+- _TR 18145 'Environmentally sustainable Artificial Intelligence'_ is een technisch rapport (TR). Het is nu "under approval".
+- Werkgroep JTC 21 heeft een "preliminary work item" genaamd _'Guidelines and metrics for the Environmental Impact of Artificial Intelligence Systems and Services'_ wat nog aan werk onderhevig is.
+
+### Welke standaarden, rapporten en specificaties zijn er al?
+Op dit moment zijn er verschillende Technische Rapporten, Technische Specificaties en Normen (onder de tabel kun je meer informatie vinden over deze termen) die al van belang zijn voor AI. In onderstaande tabel worden er enkele uitgelicht. Ben je opzoek naar meer, kijk dan eens op de [AI Standards Hub](https://aistandardshub.org) van het Verenigd Koninkrijk, of op [NEN Connect](https://connect.nen.nl).
+
+| Norm                          | Titel                                                                                         | Status                      |
+|-------------------------------|-----------------------------------------------------------------------------------------------|-----------------------------|
+| ISO/IEC 42001                | Information technology - Artificial intelligence - Management system                          | ISO Norm                    |
+| ISO/IEC TS 25058             | Systems and software engineering - Systems and software Quality Requirements and Evaluation (SQuaRE) - Guidance for quality evaluation of artificial intelligence (AI) systems | Technische Specificatie      |
+| NEN-EN-ISO/IEC 23894        | Information technology - Artificial intelligence - Guidance on risk management                | Europese Norm               |
+| NEN-EN-ISO/IEC 25059        | Software engineering - Systems and software Quality Requirements and Evaluation (SQuaRE) - Quality model for AI systems | Europese Norm               |
+| NEN-ISO/IEC 38507           | Information technology - Governance of IT - Governance implications of the use of artificial intelligence by organizations | Nederlandse Norm            |
+| NPR-CEN/CLC ISO/IEC TR 24027 | Information technology - Artificial intelligence (AI) - Bias in AI systems and AI aided decision making | Nederlandse Praktijk Richtlijn |
+| NPR-ISO/IEC TR 27563        | Security and privacy in artificial intelligence use cases - Best practices                    | Nederlandse Praktijk Richtlijn |
+| NVN-ISO/IEC TS 8200         | Information technology - Artificial intelligence - Controllability of automated artificial intelligence systems | Nederlandse Voornorm         |
+
+
+??? expander "Uitleg over gebruikte termen"
+    
+    Er zijn verschillende soorten standaarden, zoals normen, Technische Rapporten (TR) en Technische Specificaties (TS).
+
+    - Normen zijn standaarden die door standaardisatieorganisaties worden gemaakt, zoals ISO of CEN/CENELEC, of NEN.
```

Comment:
Ahh kijk, hier is een toelichting op wat normen zijn. Wellicht al eerdere vermelden in het stuk?